pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -72,13 +80,100 @@ func TestAddEntrypoint(t *testing.T) {
 	}
 }
 
+type image struct {
+	config *v1.ConfigFile
+}
+
+// RawConfigFile implements partial.UncompressedImageCore
+func (i *image) RawConfigFile() ([]byte, error) {
+	return partial.RawConfigFile(i)
+}
+
+// ConfigFile implements v1.Image
+func (i *image) ConfigFile() (*v1.ConfigFile, error) {
+	return i.config, nil
+}
+
+// MediaType implements partial.UncompressedImageCore
+func (i *image) MediaType() (types.MediaType, error) {
+	return types.DockerManifestSchema2, nil
+}
+
+// LayerByDiffID implements partial.UncompressedImageCore
+func (i *image) LayerByDiffID(diffID v1.Hash) (partial.UncompressedLayer, error) {
+	return nil, fmt.Errorf("unknown diff_id: %v", diffID)
+}
+
+func mustRawManifest(t *testing.T, img v1.Image) []byte {
+	m, err := img.RawManifest()
+	if err != nil {
+		t.Fatalf("RawManifest() = %v", err)
+	}
+	return m
+}
+
+func mustRawConfigFile(t *testing.T, img v1.Image) []byte {
+	c, err := img.RawConfigFile()
+	if err != nil {
+		t.Fatalf("RawConfigFile() = %v", err)
+	}
+	return c
+}
+
+func getImage(t *testing.T, cfg *v1.ConfigFile) v1.Image {
+	rnd, err := partial.UncompressedToImage(&image{
+		config: cfg,
+	})
+	if err != nil {
+		t.Fatalf("random.Image() = %v", err)
+	}
+	return rnd
+}
+
+func mustConfigName(t *testing.T, img v1.Image) v1.Hash {
+	h, err := img.ConfigName()
+	if err != nil {
+		t.Fatalf("ConfigName() = %v", err)
+	}
+	return h
+}
+
 func TestGetRemoteEntrypoint(t *testing.T) {
-	ep, err := entrypoint.GetRemoteEntrypoint(entrypoint.NewCache(), kanikoImage)
+	img := getImage(t, &v1.ConfigFile{
+		ContainerConfig: v1.Config{
+			Entrypoint: []string{kanikoEntrypoint},
+		},
+	})
+	expectedRepo := "kaniko"
```

Comment:
I think we should avoid using meaningful terms like "kaniko" here now that it has nothing to do with the kaniko image.